toxiproxy.go

```diff
@@ -12,7 +12,7 @@ var host string
 var port string
 
 func init() {
-	flag.StringVar(&host, "host", "localhost", "Host for toxiproxy's API to listen on")
+	flag.StringVar(&host, "host", "0.0.0.0", "Host for toxiproxy's API to listen on")
```

Comment:
This kindof seems wrong to me. Depending on where you're running it, you don't want just anyone to go and mess with your proxies.

Is there some way to just pass `-host 0.0.0.0` in the Dockerfile?